README.md

```diff
@@ -15,11 +15,12 @@ It allows you to make queries using the powerful [SQLAlchemy Core][sqlalchemy-co
 expression language, and provides support for PostgreSQL, MySQL, and SQLite.
 
 Databases is suitable for integrating against any async Web framework, such as [Starlette][starlette],
-[Sanic][sanic], [Responder][responder], [Quart][quart], [aiohttp][aiohttp], [Tornado][tornado], or [FastAPI][fastapi].
+[Sanic][sanic], [Responder][responder], [Quart][quart], [aiohttp][aiohttp], [Tornado][tornado],
```

Comment:
![image](https://user-images.githubusercontent.com/7353520/227474546-05523239-e519-4c64-829d-4eff23082945.png)

:thinking:

Reply:
Yap, got it. This were the docs. Reverting it. I can open another PR if that is the issue.